server/src/pages/shotindex/view.js

```diff
@@ -105,6 +106,12 @@ class Body extends React.Component {
     );
   }
 
+  renderSettingsPage() {
+    return (
+      <a className="button preferences" href="/settings"></a>
```

Comment:
This should have an `aria-label` on it. (Maybe other things too? It's something I only barely remember to do, but we should be aware of.)